torchmdnet/models/model.py

```diff
@@ -25,8 +25,8 @@ def create_model(args, prior_model=None, mean=None, std=None):
     -------
         nn.Module: An instance of the TorchMD_Net model.
     """
-    args["dtype"] = "float32" if "dtype" not in args else args["dtype"]
-    args["dtype"] = dtype_mapping[args["dtype"]] if isinstance(args["dtype"], str) else args["dtype"]
+    args["precision"] = 32 if "precision" not in args else args["precision"]
+    args["dtype"] = dtype_mapping[args["precision"]]
```

Comment:
Do you still need this line? You should remove the `args["dtype"]` since you already removed it from the argparser in train.py

Reply:
There needs to be still a way to send dtype down the line. Maybe I get away with just creating a local variable "dtype" in the constructor and giving that to the different modules.
Do you think this current way is problematic?